src/guides/v2.4/config-guide/prod/config-reference-systemxml.md

```diff
@@ -265,6 +265,21 @@ A `<field>`-Tag can have the following children:
 {:.bs-callout-warning}
 The nodes `more_url`, `demo_url`, `requires` and `options` are defined by a different core payment model and are only used once. These nodes are not reusable.
 
+### Field input type reference
+
+Each field has the default type of ´text´ to display its frontend input type. The following input field types can be found and used in the core at this time :
+
+| Field Type    | Description                                                                                                       |
+|---------------|-------------------------------------------------------------------------------------------------------------------|
+| ´text´        | Default. Standard text field.                                                                                     |
+| ´select´      | A common HTML select. Only one item can be selected.                                                              |
+| ´multiselect´ | A common HTML select. Multiple items can be selected at once.                                                     |
+| ´obscure´     | Commonly used for passwords. Shows asterisks as values.                                                           |
```

Comment:
Also mention that the value is encrypted and forcibly changing type using "Inspect Element" in browser will not show the value